recipes/cova/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "cova" %}
+{% set version = "0.22.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cova-{{ version }}.1.tar.gz
+    sha256: 1e8f4dd144905b71eb228dda90a99d60d16bc7f6f40a997f70416347a362ee4f
+  - url: https://files.pythonhosted.org/packages/5b/78/0c8e4b41ee425be92888bc118af7c15fc7b771a7e495d7b48b223cd6c382/sentinel-0.3.0.tar.gz
```

Comment:
Please submit this as a separate recipe.

Reply:
Do you mean for the sentinel module (line 11)? I'd rather not become the maintainer of that package. We were planning on removing our dependence on it anyway. I'll change this recipe once we do. Can I leave this PR up, or do I need to delete it and make another one when the changes are ready? Thanks.

Reply:
I merged `sentinel` from someone else some hours ago, you should be able to directly depend on it.

Reply:
Awesome, thanks! I've taken it's installation out of the recipe and put it under requirements.